compiler/luci/import/src/Importer.cpp

```diff
@@ -134,6 +135,15 @@ void convert_graph(const luci::GraphBuilderSource &source, luci::CircleReader &r
       nodefinder->enroll(i, const_node);
   }
 
+  // Create CircleVariable nodes for variable tensors
+  // TODO Add Origin if needed, skip for now
+  for (uint32_t i = 0; i < tensors.size(); ++i)
+  {
+    luci::CircleVariable *variable_node = luci::create_circlevariable(&gb_context, i);
+    if (variable_node != nullptr)
+      nodefinder->enroll(i, variable_node);
+  }
```

Comment:
If we apply above comment, I think following can be applied?

```suggestion
  for (uint32_t i = 0; i < tensors.size(); ++i)
  {
    if (auto const_node = const_builder->build(i, &gb_context))
      nodefinder->enroll(i, const_node);
    else if (auto variable_node = variable_builder->build(&gb_context, i))
      nodefinder->enroll(i, variable_node);
  }
```

Reply:
> If we apply above comment,

which comment?

> I think following can be applied?

Yes I think we can but I don't want to mix with existing code for now. You may revise with remove loop for CircleConst later.

Reply:
I recommend this not only because of better readability but also because it can prevent fragment of tensor handling logic thus we may prevent unexpected handling error.
But okay, it's your choice :)

Reply:
> it can prevent fragment of tensor handling logic

Can you explain more about this?

Reply:
It's not critical thing, just the difference of style :)

---

- If we need to change something about `tensors`, we should fix each fragmented blocks(block 1 & block 2)
- If logic of `const_builder` or `variable_builder` is changed, node can be created by both of them unintentionally.
- If other codes are inserted between the fragmented blocks, reading codes becomes little bit harder

```cpp
  // block 1
  for (uint32_t i = 0; i < tensors.size(); ++i)
  {
    auto *const_node = const_builder->build(i, &gb_context);
    if (const_node != nullptr)
      nodefinder->enroll(i, const_node);
  }

  // ...

  // block 2
  for (uint32_t i = 0; i < tensors.size(); ++i)
  {
    luci::CircleVariable *variable_node = luci::create_circlevariable(&gb_context, i);
    if (variable_node != nullptr)
      nodefinder->enroll(i, variable_node);
  }
```

Reply:
> If logic of const_builder or variable_builder is changed, node can be created by both of them unintentionally.

You've pointed an important thing which current CircleConst doesn't check.
I think CircleConst should not be created when `variable` is false.

Reply:
We can still merge two loops for creating CircleNode from tensor as you pointed.
Please post a PR after this lands if you like.